traffic_ops/traffic_ops_golang/api/alerts.go

```diff
@@ -0,0 +1,70 @@
+package api
+
```

Comment:
every other file has imports **after** leading license comment..  Can you move those so they're consistent?

Reply:
@DylanVolz I don't care if we go with the `api` package or `tostructs` we should make them all consistent

Reply:
@dangogh fixed. @dewrich I think tostructs / api refactor should be handled in a separate pr so that the refactor isn't conflated with the other changes in this pr.